code/game/machinery/telecomms/machine_interactions.dm

```diff
@@ -255,3 +255,16 @@
 	if(issilicon(user) || in_range(user, src))
 		return TRUE
 	return FALSE
+
+/obj/machinery/telecomms/multitool_act_secondary(mob/living/user, obj/item/tool)
+	var/obj/item/multitool/multitool = tool
+	var/obj/machinery/telecomms/telecomms_machine = multitool.buffer
+	if(QDELETED(multitool.buffer)||!multitool.buffer || !istype(telecomms_machine) || telecomms_machine == src) //sanity + user sanity
+		return FALSE
+	if(multitool.buffer in links)
+		to_chat(user, span_notice("Unlinked [telecomms_machine] ([telecomms_machine.id]) from the [src]."))
```

Comment:
Since they aren't going through some UI, maybe these would be better as balloon alerts

Reply:
not happening for the reasons in the previous "Requested Changes"